tests/wpt/meta/trusted-types/trusted-types-reporting-clipping-of-sample.tentative.html.ini

```diff
@@ -1,25 +1,25 @@
 [trusted-types-reporting-clipping-of-sample.tentative.html]
-  expected: TIMEOUT
+  expected: CRASH
```

Comment:
We run into https://github.com/w3c/webappsec-csp/issues/704 and https://github.com/w3c/trusted-types/issues/577 Seems like the spec is unresolved, so I think it is okay to crash for now.